### PR TITLE
Fix podspec lint

### PR DESCRIPTION
**Pull request by zladzeyka**
### Description
Fixes for tests:
- Moved AlbumsPickerViewControllerTests to the app tests with "Example Swift" as host;
- Added related Mocked tests to the app tests with "Example Swift" as host;
- In 11.4 simulator in testBoldDynamicFontGeneration() in font.with(weight: .bold, size: 14, style: .body) returns ("<UICTFont: 0x7f9158d024b0> font-family: ".SFUIText-Semibold"; font-weight: bold; font-style: normal; font-size: 14.00pt") added  if #available(iOS 12.0, *) for     func testBoldDynamicFontGeneration().

- AlbumsPickerViewController: Photo library registration needed starting from iOS 14 (PIA-1719

### Merging
Automatic

### CODE REVIEW

**Review by a-szotyori — APPROVED** [review]
Great success! 👍

File: GiniCapture/Classes/Core/GiniConfiguration.swift [hunk 1]
```diff
@@ -633,6 +633,20 @@ import UIKit
     */
     public var customMenuItems: [HelpMenuViewController.Item] = []
     
+    /**
+     Sets if the default error logger when giniErrorLoggerIsOn = true
```
Comment (a-szotyori) [inline GiniCapture/Classes/Core/GiniConfiguration.swift]:
```suggestion
     Sets the default error logger. It is only used when giniErrorLoggerIsOn is true.
```

File: GiniCapture/Classes/Networking/Extensions/GiniCaptureErrorLogger+GiniCaptureErrorLoggerDelegate.swift [hunk 2]
```diff
@@ -7,6 +7,8 @@
 
 import Foundation
 
-extension GiniCaptureErrorLogger {
-   // error.apiLibVerion())
+class GiniErrorLogger: GiniCaptureErrorLoggerDelegate {
```
Comment (a-szotyori) [inline GiniCapture/Classes/Networking/Extensions/GiniCaptureErrorLogger+GiniCaptureErrorLoggerDelegate.swift:10]: To make it print the error logs you should set an instance of this class in `GiniCapture+GiniCaptureDelegate.swift` to be the `giniErrorLogger`: `configuration.giniErrorLogger = GiniErrorLogger()`.

This way we have the error logging in place and then when the api is ready we just extend this class to send the log to the api.

**Review by a-szotyori — CHANGES_REQUESTED** [review]
Just two small requests. 😬

File: GiniCapture/Classes/Networking/Extensions/GiniCaptureErrorLogger+GiniCaptureErrorLoggerDelegate.swift (hunk 2 above)
Comment (zladzeyka) [reply GiniCapture/Classes/Networking/Extensions/GiniCaptureErrorLogger+GiniCaptureErrorLoggerDelegate.swift:10]: True, we've talked about it yesterday.

**Review by a-szotyori — APPROVED** [review]

**Pull request merged**